x11-apps/copyq/copyq-3.10.0.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils gnome2-utils
```

Comment:
... switch to *cmake.eclass* and either *xdg-utils* or *xdg.eclass* instead.

Reply:
no problem, this broke ebuild, should be good now